### PR TITLE
Fixed preview bug and radiobutton bug

### PR DESCRIPTION
**Pull request by andreasnp**
The preview bug was caused by that the store constant was placed outside of the FormFillerPreview component. It made the questionnaire only update one time even though Refero re-rendered multiple times.

The radiobutton bug was caused by the Refero-component including Toolkit, which brought with it some styling that overrided the styling in Structor. Refero does not have direct dependencies to Toolkit, but some of it's Helsenorge-packages have dependencies to Toolkit. The solution for now was to override the Toolkit styling. Toolkit is on it's way to be totally replaced with Designsystem-React, so this issue will likely solve itself.

### CODE REVIEW

**Review by kennethmyhra — APPROVED** [review]

**Comment by lassefordon** [comment]
Avvik som må fikses:
283607 flere radioknapper som fortsatt ikke vises

**Comment by lassefordon** [comment]
Remaining bugs that need to be fixed before approval:

- [ ] Task 283716 - Items disappearing from preview if conditions are added and removed

**Review by lassefordon — APPROVED** [review]

**Pull request merged**
